ui/raidboss/emulator/EventBus.ts

```diff
@@ -0,0 +1,65 @@
+type CallbackFunction = () => void;
+type EventMapEntry = {
+  event: string;
+  // Disable ban-types due to scope being any object type
+  // eslint-disable-next-line @typescript-eslint/ban-types
+  scope: object;
+  callback: CallbackFunction;
+};
```

Comment:
```suggestion
// Disable ban-types due to scope being any object type
// eslint-disable-next-line @typescript-eslint/ban-types
type Scope = object;

type EventMapEntry = {
  event: string;
  scope: Scope;
  callback: CallbackFunction;
};
```

I think better to not repeat the disable everywhere and just keep the "problematic" type in one place.